pkg/controllers/management/aks/aks_cluster_handler_mockc_test.go

```diff
@@ -1,26 +1,28 @@
-//
 // CODE GENERATED AUTOMATICALLY WITH github.com/kelveny/mockcompose
-// THIS FILE SHOULD NOT BE EDITED BY HAND
-//
+// But has since been manually updated.
```

Comment:
golangci-lint ignores this file because of the comment `CODE GENERATED AUTOMATICALLY`. If I delete this line, then golangci-lint wants me to refactor the file. So I left the original comment to reduce the blast radius of changes.

Reply:
We could use the `nolint` comment instead or exclude in the golangci-lint config file.

Reply:
Used nolint in this separate follow up PR. https://github.com/rancher/rancher/pull/41768/commits/345915e86e5405d943252577fa400807bbe6788a